File: check_logs.js
const { NodeSSH } = require('node-ssh')
const ssh = new NodeSSH()

async function run() {
  await ssh.connect({ host: '62.72.32.37', username: 'root', password: 'Aa7161062.123' })
  const r = await ssh.execCommand('tail -n 100 /var/log/nginx/access.log | grep django-admin')
  console.log(r.stdout)
  process.exit(0)
}
run()

File: check_logs.py
import paramiko
import sys

def run():
    client = paramiko.SSHClient()
    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
    client.connect('62.72.32.37', username='root', password='Aa7161062.123')
    stdin, stdout, stderr = client.exec_command('tail -n 100 /var/log/nginx/access.log | grep django-admin')
    print(stdout.read().decode())
    client.close()

if __name__ == "__main__":
    run()
